doc/conf.py

```diff
@@ -81,7 +81,7 @@
 
 # General information about the project.
 project = u'Apache Traffic Server'
-copyright = u'2016, dev@trafficserver.apache.org'
+copyright = u'2017, dev@trafficserver.apache.org'
```

Comment:
Sounds like a good idea to me. I did a quick look into the legal aspects of this and it seems fine.